lib/MiLight/CctPacketFormatter.cpp

```diff
@@ -37,11 +54,24 @@ void CctPacketFormatter::updateTemperature(uint8_t value) {
 }
 
 void CctPacketFormatter::command(uint8_t command, uint8_t arg) {
+  uint8_t checksum;
+  uint8_t i;
+
   pushPacket();
   if (held) {
     command |= 0x80;
   }
   currentPacket[CCT_COMMAND_INDEX] = command;
+
+  // Calculate checksum over packet length .. sequenceNum
+  checksum = 7; // Packet length is not part of packet
+  for (i = 0; i < 6; i++) {
```

Comment:
Unless `i` is going to be used outside of the scope of the loop (doesn't look like it will), could we initialize it here?

Reply:
I assume you prefer this code instead?

```
for (uint8_t i = 0; i < 6; i++) {
```

That's fine.